microstream-rest/src/test/groovy/io/micronaut/microstream/rest/RestConfigurationSpec.groovy

```diff
@@ -0,0 +1,114 @@
+package io.micronaut.microstream.rest
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import io.micronaut.context.ApplicationContext
+import io.micronaut.http.HttpStatus
+import io.micronaut.http.client.BlockingHttpClient
+import io.micronaut.http.client.HttpClient
+import io.micronaut.http.client.exceptions.HttpClientResponseException
+import io.micronaut.runtime.server.EmbeddedServer
+import spock.lang.Shared
+import spock.lang.Specification
+import spock.lang.TempDir
+
+class RestConfigurationSpec extends Specification {
+
+    @TempDir
+    @Shared
+    File tempDir
+
+    def "prefix can be configured"() {
```

Comment:
use `void`

Reply:
done (as before it's just more bytes on disk)